Gruntfile.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(grunt) {
         uglify: {
           my_target: {
             files: {
-              'dist/Leaflet.BlurredLocation.js': ['src/location_tags.js', 'src/locationForm.js', 'main.js']
+              'dist/Leaflet.BlurredLocation.js': ['src/location_tags.js', 'main.js']
```

Comment:
Hmm, so here, are you concatenating these files into `dist/Leaflet.BlurredLocation.js`, and how does this work before or after `browserify` compiles into `dist/Leaflet.BlurredLocation.js`? Don't you want Browserify to do the compiling, then uglify the output? I'm actually not that familiar with uglify - is that typically run after Browserify?